api/explored/client.go

```diff
@@ -40,6 +44,127 @@ func (c *Client) ConsensusTip() (resp ConsensusTipResponse, err error) {
 	return
 }
 
+// ExplorerSiacoinElement gets the Siacoin element with the given ID.
+func (c *Client) ExplorerSiacoinElement(id types.ElementID) (resp types.SiacoinElement, err error) {
+	data, err := json.Marshal(id)
+	if err != nil {
+		return
+	}
+	err = c.c.Get(fmt.Sprintf("/api/explorer/element/siacoin/%s", string(data)), &resp)
+	return
+}
+
+// ExplorerSiafundElement gets the Siafund element with the given ID.
+func (c *Client) ExplorerSiafundElement(id types.ElementID) (resp types.SiafundElement, err error) {
+	data, err := json.Marshal(id)
+	if err != nil {
+		return
+	}
+	err = c.c.Get(fmt.Sprintf("/api/explorer/element/siafund/%s", string(data)), &resp)
+	return
+}
+
+// ExplorerFileContractElement gets the file contract element with the given ID.
+func (c *Client) ExplorerFileContractElement(id types.ElementID) (resp types.FileContractElement, err error) {
+	data, err := json.Marshal(id)
+	if err != nil {
+		return
+	}
+	err = c.c.Get(fmt.Sprintf("/api/explorer/element/contract/%s", string(data)), &resp)
+	return
+}
+
+// ExplorerChainStats gets stats about the block at the given index.
+func (c *Client) ExplorerChainStats(index types.ChainIndex) (resp explorer.ChainStats, err error) {
+	data, err := json.Marshal(index)
+	if err != nil {
+		return
+	}
+	err = c.c.Get(fmt.Sprintf("/api/explorer/chain/stats/%s", string(data)), &resp)
+	return
+}
+
+// ExplorerChainStatsLatest gets stats about the latest block.
+func (c *Client) ExplorerChainStatsLatest() (resp explorer.ChainStats, err error) {
+	err = c.c.Get("/api/explorer/chain/stats/latest", &resp)
+	return
+}
+
+// ExplorerSearch gets information about a given element.
+func (c *Client) ExplorerSearch(id types.ElementID) (resp ExplorerSearchResponse, err error) {
+	data, err := json.Marshal(id)
+	if err != nil {
+		return
+	}
+	err = c.c.Get(fmt.Sprintf("/api/explorer/search/%s", string(data)), &resp)
+	return
+}
+
+// ExplorerBalance gets the siacoin and siafund balance of an address.
+func (c *Client) ExplorerBalance(address types.Address) (resp ExplorerWalletBalanceResponse, err error) {
```

Comment:
`ExplorerAddressBalance`

(a good rule of thumb is that the method name should match the API route)